CHANGELOG.mdx

```diff
@@ -17,6 +17,13 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.0] - 2021-07-08
```

Comment:
Hmm seems we are missing a changelog for UnexpecTED?

Reply:
This is rasa-sdk 😅

Reply:
haha oh yeah 🤦